server-mirror-git/src/main/java/com/linecorp/centraldogma/server/internal/mirror/AbstractGitMirror.java

```diff
@@ -257,7 +257,8 @@ void mirrorRemoteToLocal(
             changes.put(mirrorStatePath, Change.ofJsonUpsert(
                     mirrorStatePath, "{ \"sourceRevision\": \"" + headCommitId.name() + "\" }"));
             // Construct the log message and log.
-            summary = "Mirror " + abbrId + ", " + remoteRepoUri() + '#' + remoteBranch() +
+            summary = "Mirror " + abbrId + ", " + remoteRepoUri() + '#' +
+                    (remoteBranch() != null ? '#' + remoteBranch() : "") +
```

Comment:
It seems that double `#` is added.
```suggestion
            summary = "Mirror " + abbrId + ", " + remoteRepoUri() +
                    (remoteBranch() != null ? '#' + remoteBranch() : "") +
```

Reply:
Yes, that's correct. Thanks 🙇